### ISSUE TITLE
Using vertical levels with weather model initialization

### ISSUE THREAD

**Issue by yoelchetboun** [firelab/windninja#510]
Hi! 

I apologize if my question is stupid, but I would like to calculate the wind speed at 80m height using 'weather model initialization'. Indeed, I have several wind speed heights in my netcdf file that I'd like to use (10m, 40m, 90m, 100m...) and I saw in the tutorial 4 that the ability to use information at all vertical levels will be added shortly? 

Is this feature available?

Thank you again!
Regards,

**Comment by nwagenbrenner** [comment]
@yoelchetboun WindNinja uses the 10-m wind by default for initialization. Initial testing with 3D input winds did not provide much benefit over using just the 10-m winds as input, but noticeably increased model run time due to the required 3D interpolations. It also required longer download times since the full 3D model data must be downloaded, rather than just the surface data. For this reason, we haven't made the 3D initialization the default. If you are familiar with building from source and are running on Linux, however, you can turn on the CMake option `NOMADS_ENABLE_3D` to have WindNinja use the full 3D winds for initialization. Note that this functionality only works with WRF-ARW  NetCDF output and possibly one or two of the NOMADS forecasts.

**Comment by yoelchetboun** [comment]
Thank you for your very quick anwer! I understand, unfortunately In my case I don't have a full WRF output but only a few 2D layers. 

Maybe, the best solution is to use only my 90-m wind as wx input in order to be as close as possible to my target height of 80-m? In that case, is there any way of telling WindNinja that my initialization data is at 90-m height instead of 10m? Is using the 'input_wind_height' option is the right way to do it?

Thank you again!!

**Comment by nwagenbrenner** [comment]
@yoelchetboun Yes, that is probably a good option for you. You can use the griddedInitialization option in WindNinja. This option isn't fully documented as it was originally implemented for internal use. We are working to update our documentation to include this feature. For now, you can see the attached for an example of how to use this option. It should do exactly what you are requesting.

[griddedInit.zip](https://github.com/firelab/windninja/files/13788379/griddedInit.zip)
